### PR TITLE
EVAKA_FIX: varda paatos hakemuspvm

### PR DESCRIPTION
**Pull request by hruoho**
#### Summary

Use application.sentdate when possible

### CODE REVIEW

File: service/src/main/kotlin/fi/espoo/evaka/varda/VardaOrganizerChild.kt [hunk 1]
```diff
@@ -79,7 +79,7 @@ private fun createVardaPersonAndChild(
 ): Long {
     val personPayload = getVardaPersonPayload(tx, evakaPersonId, organizerOid)
 
-    val vardaPerson = client.createPerson(personPayload) ?: error("Couldn't create Varda person (nor child)")
+    val vardaPerson = client.createPerson(personPayload) ?: error("VardaUpdate: Couldn't create Varda person $personPayload")
```
Comment (kimifoud) [inline service/src/main/kotlin/fi/espoo/evaka/varda/VardaOrganizerChild.kt:82]: Is it fine that ssn gets logged?

Comment (hruoho) [reply service/src/main/kotlin/fi/espoo/evaka/varda/VardaOrganizerChild.kt:82]: It will be replaced with `REDACTED-SSN` in logs

Comment (jaahon) [reply service/src/main/kotlin/fi/espoo/evaka/varda/VardaOrganizerChild.kt:82]: Ssns should be filtered by our log configs.

Comment (jaahon) [reply service/src/main/kotlin/fi/espoo/evaka/varda/VardaOrganizerChild.kt:82]: I should refresh the page before answering 😄

Comment (kimifoud) [reply service/src/main/kotlin/fi/espoo/evaka/varda/VardaOrganizerChild.kt:82]: Ah, nice!

**Review by kimifoud — APPROVED** [review]

**Pull request merged**